beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -853,3 +854,19 @@ func (bs *Server) GetBlockRoot(w http.ResponseWriter, r *http.Request) {
 	}
 	http2.WriteJson(w, response)
 }
+
+// GetDepositContract retrieves deposit contract address and genesis fork version.
+func (_ *Server) GetDepositContract(w http.ResponseWriter, r *http.Request) {
+	_, span := trace.StartSpan(r.Context(), "beacon.GetDepositContract")
+	defer span.End()
+
+	http2.WriteJson(w, &DepositContractResponse{
+		Data: &struct {
```

Comment:
would it be better to just have a type to not potentially mess up the duplicated json naming

Reply:
I had originally done that with `/eth/v1/beacon/blocks/{block_id}/root` but as the type wasn't being used by anything else, we opted to use a nested struct. I don't think I need to duplicate the json tags so I could just add them when I pass the data and remove them from the original struct type declaration.